components/atoms/details2/components/header.js

```diff
@@ -31,15 +33,16 @@ export const Details2Header = ({
           />
         )}
 
-        <Text size={SIZE.MD} context='dark' content={title} />
+        <Text size={SIZE.MD} context={titleContext || 'dark'} content={title} />
```

Comment:
Use defaultProps instead of `|| dark`

Reply:
Did you fix this? @mohsenParvar

Reply:
@majid-amiri it is fixed

Reply:
@mohsenParvar Why I can't see the change? I still see `context={titleContext || 'dark'}`